sia/upload_test.go

```diff
@@ -39,4 +56,10 @@ func testUploadFile(t *testing.T, c *Core) {
 	if c.hostDB.Size() < 1 {
 		t.Fatal("Hostdb got pruned while trying to make a contract?")
 	}
+
+	// Check that the file can be downloaded.
+	err = c.renter.Download("one", "renterDownload")
+	if err != nil {
+		t.Error(err)
+	}
```

Comment:
Do we leave it in the make clean though?

Should we just consolidate to one directory? I feel like if you're going to inspect it at all, it should be in both the .gitignore and in the `make clean` command, even if it's normally removed automatically.